docker/android-sdk/tools/watch

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+inotifywait -mr $HOME/.android -e create -e moved_to -e modify -e attrib |
```

Comment:
we can but we won't be able to support a common debug keystore file for all builds (unless we include it in the slave image with a `ONBUILD` directive)